message_sender/serializers.py

```diff
@@ -1,34 +1,20 @@
 from .models import Inbound, Outbound, OutboundSendFailure, Channel
 from rest_hooks.models import Hook
 from rest_framework import serializers
-from rest_framework.exceptions import APIException
-
-
-class IncorrectChannel(APIException):
-    status_code = 400
-    default_detail = 'The channel supplied is not configured.'
-    default_code = 'incorrect_channels'
 
 
 class OutboundSerializer(serializers.HyperlinkedModelSerializer):
 
+    channel = serializers.PrimaryKeyRelatedField(
+        queryset=Channel.objects.all(), required=False)
```

Comment:
It is on run time, I just created a test that adds a outbound then a new channel, then another outbound on the new channel and it works.